src/apirequest.ts

```diff
@@ -307,3 +308,14 @@ class ProgressStream extends stream.Transform {
     callback();
   }
 }
+
+interface Headers {
+  // tslint:disable-next-line no-any
+  [index: string]: any;
+}
+
+function populateAPIHeader(headers: Headers) {
+  const pkg = require('../../package.json');
+  const nodeVersion = process.version.replace(/^v/, '');
+  headers['x-goog-api-client'] = `gdcl/${pkg.version} gl-node/${nodeVersion}`;
```

Comment:
There's a bunch of code here that is also doing user agent stuff ....  what should happen to it?
https://github.com/googleapis/nodejs-googleapis-common/blob/master/src/apirequest.ts#L243

Reply:
There is also code here that does some stuff, we should check out:
https://github.com/googleapis/google-auth-library-nodejs/blob/master/src/transporters.ts#L63

I am worried that we're going to try to send the data both in the user agent header *and* in this bonus header.  In that case - I have no idea who wins.  @dazuma ?

Reply:
Side note @bcoe - would be worth adding a unit test that verifies we *don't* send anything in the user agent field that would conflict with this data.

Reply:
I'd rather remove the `User-Agent` fields than add unit tests for the old approach. From my reading of the document, it does say that `x-goog-api-client` should be preferred as a header.

Reply:
Yes!  I'm saying in this PR, let's stop sending version data in the user agent field.  Then let's add a test to make sure we aren't :)

Reply:
In discussion with Daniel, it sounds like we can leave `User-Agent`, and `x-goog-api-client` will take precedence. This is preferable because we aggressively populate the `User-Agent` in a variety of places:

1. googleapis-common.
2. if it's not present here, in google-auth-library.
3. if it's not present here, in node-fetch itself.